server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java

```diff
@@ -417,9 +452,11 @@ private boolean checkTTL(long time) {
   }
 
   private void insertBatchToTsFileProcessor(BatchInsertPlan batchInsertPlan,
```

Comment:
add a comment on the top of this method: thread-safety that the caller need to guarantee

Reply:
added~